internal/rootcoord/root_coord.go

```diff
@@ -560,6 +558,7 @@ func (c *Core) Init() error {
 		log.Info("RootCoord enter standby mode successfully")
 	} else {
 		c.initOnce.Do(func() {
+			c.UpdateStateCode(commonpb.StateCode_Initializing)
```

Comment:
Removing the initializing state before reloading the meta table may result in some requests being accepted and subsequently failing?

Reply:
I noticed you removed the init state from the standby model.

Reply:
Actually no place check this states, but just mark it as unhealthy.